setupJest.js

```diff
@@ -11,7 +10,7 @@ configure({
     seleniumUrl: `http://localhost:9515`,
   },
   webdriverOptions: {
-    baseUrl: `http://localhost:${devServerPort}`,
+    baseUrl: `http://localhost:9615`,
```

Comment:
Actually why hard-code this instead of keep using `devServerPort` from `scripts/config`?

Reply:
I saw this is answered in the comments in the upstream change. LGTM for now then